### DISCUSSION TITLE
"No FAT32 formatted drive found, unable to archive nwipe file, Wildcat Point-LP USB"

### DISCUSSION THREAD

**Discussion by thebearnecessities** [PartialVolume/shredos.x86_64#224] [General]
Anyone has this error before?  Not sure why its suddenly saying that when I've been using this shredOS usb stick regularly, perfectly fine for a couple months.

It have a fairly standards setup, trying to wipe some quite old laptops, using a (FAT32 formatted!) usb stick created in rufus, latest version etc.

I wouldn't normally mind about the log file, but I do need the PDF's which it seems are not being archived either.

Hope this is the correct place to ask this question, apologies if not, great software, much appreciated.

**Comment by PartialVolume** [comment]
I would switch to the virtual terminal ALT F2 and run 'fdisk -l' and see if the OS is actually seeing the USB flash drive. Unplug it and plug it in a different port and rerun fdisk -l and see if it appears. It's ok to unplug it, the PDFs will still exist on the RAM disc.

What's the make & model?

**Reply by thebearnecessities** [reply]
its happened on two, the one in front of me is a lenovo yoga 500, the other one i dont have here with me right now, but its a HP something, HP ProBook I think.

I'll try the fdisk stuff just now, much appreciated.

**Reply by thebearnecessities** [reply]
Running fdisk -l showed only the HDD, but then using a different USB port showed me the HDD and the USB.

But it wouldnt let me ALTF2 back to nwipe, and so i had to reboot to get into nwipe, then I then ran nwipe, waited a minute and cancelled out and Im getting the same error "No FAT32 formatted drive found, unable to archive nwipe file".

Tried rebooting and running fdisk -l, and trying all three usb ports but fdisk is just seeing the HDD now everytime.

I've attached the lspci photo

(this is all on the lenovo yoga 500 btw )

![IMG_20240223_132919](https://github.com/PartialVolume/shredos.x86_64/assets/1389872/a1fbd13d-09ac-4eec-bd27-e7a25c1749cb)


(NB. as nwipe starts up on boot, it gives "unable to create qlaxxx trace instance" but now sure if thats relevent)

**Comment by PartialVolume** [comment]
Ah, HP, I might have guessed. Can you run `lspci` as well and take a photo and post it. Thanks.

**Reply by thebearnecessities** [reply]
new reply (with 5 edits!) above

**Comment by PartialVolume** [comment]
Are there any USB settings in the bios? On some systems you can disable the USB ports altogether or for certain speeds and to disable power to USB. Is it running on mains power or battery? I just checked the drivers for Intel Wildcat LP Point USB and it looks  like they are loaded. 

>But it wouldnt let me ALTF2 back to nwipe, and so i had to reboot to get into nwipe, then I then ran nwipe, waited a minute and cancelled out and Im getting the same error "No FAT32 formatted drive found, unable to archive nwipe file".

ALT F1 takes you back to the default nwipe screen. ALT F2 and ALT F3 are alternate terminals.

I was just wondering whether the power is failing on the USB ports. If you connect something like a phone to the USB port does it charge?

**Reply by thebearnecessities** [reply]
![IMG_20240223_171028](https://github.com/PartialVolume/shredos.x86_64/assets/1389872/86a6b8fb-6fc1-4086-9d62-7d1a928d0f36)
B
![IMG_20240223_171013](https://github.com/PartialVolume/shredos.x86_64/assets/1389872/5f43e34b-80d0-4624-888d-ca902b6a06bd)

Not much in the bios, all three usb ports charge my phone ok.

It was running on mains battery yeah, but I did try it without too, but still can't get it to save the pdf.

Not sure what to do next.  Is there just a certain % of laptops where the software won't work?

**Comment by PartialVolume** [comment]
Not so much a percentage just certain hardware, hence the importance of the lspci output. It isn't the first time that Intel Wildcat LP Point USB chipset has been an issue. So could quite possibly be a buggy driver.

If USB isn't working for saving the PDFs, then your only option is to ftp the PDFs to a ftp server on your local network. Or maybe to a public ftp server, maybe like ftp://ftp.dlptest.com/. I've never tried a public ftp server though so you are on your own with that.

I just tried to search https://github.com/PartialVolume/shredos.x86_64 for wildcat but Google failed to find anything most likely because most lspci is uploaded as a photo. I think I may have to extract the text from these snapshots, like so, so I can find past references: 

sh-5.2# Ispci
00:00.0 Host bridge: Intel Corporation Broadwell-U Host Bridge -OPI (rev 08) 00:02.0 VGA compatible controller: Intel Corporation HD Graphics (rev 08)
00:03.0 Audio device: Intel Corporation Broadwell-U Audio Controller (rev 08)
00:14.0 USB controller: Intel Corporation Wildcat Point-LP USB xHCI Controller (rev 03)
00:16.0 Communication controller: Intel Corporation Wildcat Point-LP MEI Controller # 1 (rev 03) 00:16.0 Audio device: Intel Corporation Wildcat Point-LP High Definition Audio Controller (rev 03)
00:1c.0 PCI bridge: Intel Corporation Wildcat Point-LP PCI Express Root Port # 1 (rev e3)
00:1c.2 PCI bridge: Intel Corporation Wildcat Point-LP PCI Express Root Port # 3 (rev e3)
00:1c.3 PCI bridge: Intel Corporation Wildcat Point-LP PCI Express Root Port # 4 (rev e3)
00:1d.0 USB controller: Intel Corporation Wildcat Point-LP USB EHCI Controller (rev 03)
00:15.0 ISA bridge: Intel Corporation Wildcat Point-LP LPC Controller (rev 03)
00:10.2 SATA controller: Intel Corporation Wildcat Point-LP SATA Controller [AHCI Model (rev 03)
00:11.3 SMBus: Intel Corporation Wildcat Point-LP SMBus Controller (rev 03)
00:15.6 Signal processing controller: Intel Corporation Wildcat Point-LP Thermal Management Controller (rev 83) 02:00.0 Network controller: Intel Corporation Wireless 3160 (reu 93)
03:00.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controlle
sh-5.2# 

lenovo

**Comment by PartialVolume** [comment]
I knew Wildcat USB had popped up before. See #211

**Comment by PartialVolume** [comment]
See https://forums.linuxmint.com/viewtopic.php?t=375551 same issue, fixed by a kernel upgrade. So it's certainly possible it may just be a bug in the current version of the driver and it will work in the next upgrade.

**Reply by ITTTJ** [reply]
When do you please expect the update will be released?

**Comment by PartialVolume** [comment]
Also see #139 (Lenovo ThinkCentre M920q) A Bios update fixed the non functioning USB ports.

**Comment by PartialVolume** [comment]
I'm hoping to do a full update, new kernel/drivers and latest nwipe 0.36 sometime in March but depends on other commitments.
